content/W2016/W16-event-gitworkshop.md

```diff
@@ -0,0 +1,36 @@
+Title: Technical Workshop: Intro to Git and Open-Source
+Date: 2016-03-03 14:00
+Category: Events
+Tags: Blackberry, workshop, git, open-source
+Slug: git-workshop-w16
+Author: Fatema Boxwala
+Summary: Join us for a fun workshop to learn the basics of Git, Github and 
+open-source contributions!
+
+What is Git? Why are people always telling me to make branches? How does one 
+contribute to an open-source project? We’ll be answering all these questions
+and more at our Intro to Git and Open-Source Workshop! Fatema will (try) to 
+explain the basics of Git, and by the end of the workshop all participants will
+have submitted a pull request to a public repository. Upper year mentors will
+be around to answer questions and dinner will be provided!
+
+This workshop is recommended for beginners, you can view the repo we’ll be working
+on [here](https://github.com/fboxwala/git-workshop-W16). Navigate to the wiki
+tab to view the full curriculum. Space for the workshop is limited so make 
+sure to [register](https://docs.google.com/forms/d/1eoRTSp2CtEHkbp8yIoxnMSfC_0I5w2EDvHocD4aoL7Y/viewform)
+soon!
+
+Technical Workshop Series: This event is generously sponsored by BlackBerry,
+as part of our Technical Workshop Series. We are hosting three such workshops
+focused on teaching technical skills in partnership with BlackBerry. To learn 
+more about our Technical Workshop Series and partnership with BlackBerry see 
+our [sponsors page]({filename}/pages/sponsors.md)
+
+Note: This is a [women-only](http://wics.uwaterloo.ca/faq) event.
+
+## Event Details ##
+
++ **Who:** [Women-only](http://wics.uwaterloo.ca/faq)
++ **What:** Git Workshop
++ **Where:** STC 0050
++ **When:** Tuesday March 15, 5:30&ndash;7:30PM
```

Comment:
pls do. I designed it to be pretty short, a one night thing. you can take a look and let me know if you think 2 hours wont be enough.